HLTrigger/Configuration/python/HLT_75e33/eventsetup/hltESPPixelCPEFastParams_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+
+def _addProcessPixelCPEFastParamsPhase2(process):
+    process.hltESPPixelCPEFastParamsPhase2 = cms.ESProducer('PixelCPEFastParamsESProducerAlpakaPhase2@alpaka', 
+    ComponentName = cms.string("PixelCPEFastParamsPhase2"),
+    appendToDataLabel = cms.string(''),
+    alpaka = cms.untracked.PSet(backend = cms.untracked.string(''))
+)
+
```

Comment:
pedantic suggestion:
```suggestion
def _addProcessPixelCPEFastParamsPhase2(process):
    process.hltESPPixelCPEFastParamsPhase2 = cms.ESProducer('PixelCPEFastParamsESProducerAlpakaPhase2@alpaka', 
        ComponentName = cms.string("PixelCPEFastParamsPhase2"),
        appendToDataLabel = cms.string(''),
        alpaka = cms.untracked.PSet(backend = cms.untracked.string('')
    )
)
```